### PR TITLE
Do not send non existing files to qiita

### PR DESCRIPTION
**Pull request by josenavas**

### CODE REVIEW

File: qp_deblur/deblur.py [hunk 1]
```diff
@@ -129,14 +131,31 @@ def deblur(qclient, job_id, parameters, out_dir):
...
+    final_biom = pb('final.biom')
+    final_seqs = pb('final.seqs.fa')
+    final_biom_16s = pb('final.only-16s.biom')
+    final_seqs_na = pb('final.seqs.fa.no_artifacts')
+
+    if not exists(final_biom_16s):
+        # Create an empty table. We need to send something to Qiita that is
+        # a valid BIOM, so we are going to create an empty table
+        t = Table([], [], [])
+        with biom_open(final_biom_16s, 'r+') as f:
```
Comment (ElDeveloper) [inline qp_deblur/deblur.py]: 'r+'?  Seems odd. But I guess if it works ` ¯\_(ツ)_/¯`

**Comment by ElDeveloper** [comment]
👍

File: qp_deblur/deblur.py (hunk 1 above)
Comment (josenavas) [inline qp_deblur/deblur.py]: Yeah, I started using r+ when dealing with hdf5 files due to this: http://docs.h5py.org/en/latest/high/file.html#File.mode

Comment (ElDeveloper) [inline qp_deblur/deblur.py]: Interesting, don't we want to just write the file? I almost want to say that we want to do w-.

This is a minor digression, shouldn't be a blocker to merge if someone else approves this PR.

**Pull request merged**